### ISSUE TITLE
Youtube videos does not play GV+FFTV

### ISSUE THREAD

**Issue by psymoon** [mozilla-mobile/firefox-tv#1915]
## Steps to reproduce
- gecko variant
- go to Youtube.com TV
- play any video

### Expected behavior
- video plays successfully

### Actual behavior
- video does not play

### Device information
* Fire TV device: AFTT
* Latest affected Firefox version: master
* Earliest affected Firefox version: ?

**Comment by devinreams** [comment]
Ah, so as I understand it (and to help cross-link) this is beyond the issue where autoplay is blocked: #1901

**Comment by psymoon** [comment]
Confirmed video plays in gecko-release v65

Possible cause (codec stack trace):
```03-07 12:50:56.852 16115 16129 E GeckoRemoteCodec: codec creation error
03-07 12:50:56.852 16115 16129 E GeckoRemoteCodec: android.media.MediaCodec$CodecException: Failed to initialize OMX.amlogic.vp9.decoder.awesome, error 0xfffffff4
03-07 12:50:56.852 16115 16129 E GeckoRemoteCodec:     at android.media.MediaCodec.native_setup(Native Method)
03-07 12:50:56.852 16115 16129 E GeckoRemoteCodec:     at android.media.MediaCodec.<init>(MediaCodec.java:1777)
03-07 12:50:56.852 16115 16129 E GeckoRemoteCodec:     at android.media.MediaCodec.createByCodecName(MediaCodec.java:1758)
03-07 12:50:56.852 16115 16129 E GeckoRemoteCodec:     at org.mozilla.gecko.media.LollipopAsyncCodec.<init>(LollipopAsyncCodec.java:140)
03-07 12:50:56.852 16115 16129 E GeckoRemoteCodec:     at org.mozilla.gecko.media.AsyncCodecFactory.create(AsyncCodecFactory.java:15)
03-07 12:50:56.852 16115 16129 E GeckoRemoteCodec:     at org.mozilla.gecko.media.Codec.configureCodec(Codec.java:445)
03-07 12:50:56.852 16115 16129 E GeckoRemoteCodec:     at org.mozilla.gecko.media.Codec.configure(Codec.java:398)
03-07 12:50:56.852 16115 16129 E GeckoRemoteCodec:     at org.mozilla.gecko.media.ICodec$Stub.onTransact(ICodec.java:75)
03-07 12:50:56.852 16115 16129 E GeckoRemoteCodec:     at android.os.Binder.execTransact(Binder.java:565)
03-07 12:50:56.993 16115 16128 E GeckoRemoteCodec: codec creation error
03-07 12:50:56.993 16115 16128 E GeckoRemoteCodec: android.media.MediaCodec$CodecException: Failed to initialize OMX.amlogic.vp9.decoder.awesome, error 0xfffffff4
03-07 12:50:56.993 16115 16128 E GeckoRemoteCodec:     at android.media.MediaCodec.native_setup(Native Method)
03-07 12:50:56.993 16115 16128 E GeckoRemoteCodec:     at android.media.MediaCodec.<init>(MediaCodec.java:1777)
03-07 12:50:56.993 16115 16128 E GeckoRemoteCodec:     at android.media.MediaCodec.createByCodecName(MediaCodec.java:1758)
03-07 12:50:56.993 16115 16128 E GeckoRemoteCodec:     at org.mozilla.gecko.media.LollipopAsyncCodec.<init>(LollipopAsyncCodec.java:140)
03-07 12:50:56.993 16115 16128 E GeckoRemoteCodec:     at org.mozilla.gecko.media.AsyncCodecFactory.create(AsyncCodecFactory.java:15)
03-07 12:50:56.993 16115 16128 E GeckoRemoteCodec:     at org.mozilla.gecko.media.Codec.configureCodec(Codec.java:445)
03-07 12:50:56.993 16115 16128 E GeckoRemoteCodec:     at org.mozilla.gecko.media.Codec.configure(Codec.java:398)
03-07 12:50:56.993 16115 16128 E GeckoRemoteCodec:     at org.mozilla.gecko.media.ICodec$Stub.onTransact(ICodec.java:75)
03-07 12:50:56.993 16115 16128 E GeckoRemoteCodec:     at android.os.Binder.execTransact(Binder.java:565)```

Related bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1533252

**Comment by cpeterso** [comment]
I filed a new Gecko bug and noted that this appears to be a regression in GV 66 or 67:
https://bugzilla.mozilla.org/show_bug.cgi?id=1533511

**Comment by jesup** [comment]
Note that :media processes keep dying (and restarting; sometimes this ends up playing, other times you end up with a black screen)

[foo.txt](https://github.com/mozilla-mobile/firefox-tv/files/2963125/foo.txt)

Sigh, attach files in github insists they have filetype-suffixes....

**Comment by psymoon** [comment]
Confirmed fixed with nightly 68.0.20190405095028
